packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -179,17 +190,29 @@ $image-tile-text-transform: translate3d(5px, 0, 0);
     left: 0;
     right: 0;
     bottom: 0;
+
+    &:focus {
+        @include image-tile-focus();
+    }
 }
 
 .c-imageTile-link--toggle {
     display: block;
     position: static;
     pointer-events: none;
+
+    &:focus {
+        outline: none;
+    }
 }
```

Comment:
This component has two states - link and toggle. When in toggle state I am removing focus styles for the anchor. Focus styles are added for the toggle state, they are just not needed here.